index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
Your header section layout also does not match the Figma template. Kindly fix it. Consider using the same font size, font weight, padding and margins as in the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177908878-3bd7c4ab-c6a2-4394-adaa-ff6340283dd7.png) | ![image](https://user-images.githubusercontent.com/33268927/177908930-a9e235d9-e7a1-46e5-973c-1ffea12d3252.png) |